app/js/actions/orderAction.js

```diff
@@ -14,3 +14,7 @@ export const setSelectedOrder = orderDetails => ({
   type: SET_SELECTED_ORDER,
   ...orderDetails,
 });
+
+export const removeSelectedOrder = () => ({
```

Comment:
you're dispatching `removeSelectedOrder` but I see just null values been set in state to null which doesn't really depicting a remove operation, don't you think it would be cool to just dispatch `SET_SELECTED_ORDER` which is already available and then pass the required arguments directly from the component ?

Reply:
Yp @rhenshaw56 
Using the already existing `SET_SELECTED_ORDER` can actually help me achieve the same functionality. I never thought about using that approach.
Thanks for the feedback.

Reply:
I have updated the code implementing you feedback @rhenshaw56.
Thanks again.